crates/containerd-shim-wasm/src/lib.rs

```diff
@@ -5,7 +5,9 @@
 
 pub mod sandbox;
 
+mod macros;
 pub mod services;
+pub mod sys;
```

Comment:
I think using the [`path` attribute](https://doc.rust-lang.org/reference/items/modules.html#the-path-attribute) combined with [`cfg_atr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) would be cleaner:
```rust
#[cfg_attr(unix, path = "sys/unix.rs")]
#[cfg_attr(windows, path = "sys/windows.rs")]
mod sys;
```
With that I think you can get rid of `cfg_windows` / `cfg_unix` macros.

Reply:
I've updated and I think it does simplify things! I did something similar on the instance as well.

We did loose the granular modules (but could get them back if wanted with a few tweaks) https://github.com/containerd/runwasi/pull/238#discussion_r1300726884